src/App.js

```diff
@@ -1,12 +1,21 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
As part of JS best practices, kindly do not disable the linters since they are there to help you produce the best and standard code. Kindly consider removing this line and try to solve the real problem instead of disabling the linters. You can check this [thread](https://stackoverflow.com/questions/67677676/eslint-import-no-extraneous-dependencies-error-on-path) for an idea on how to resolve it.